README.md

```diff
@@ -263,8 +263,29 @@ $ cosa init https://github.com/coreos/fedora-coreos-config.git
 $ cosa fetch && cosa build
 ```
 
-#### Using different CA certificates
+#### Running the most recent CoreOS image
+
+You have built a qcow using the assembler and now you want to
+easily run it? You can use `cosa run` for that. It will start up
+a VM using qemu and automatically log you in on the serial console
+of the machine. The login message will also tell you how to destroy
+the VM.
+
+```
+$ cosa build
+$ cosa run
+```
 
+Want SSH access? We can pass port 2222 from the host into the
+container and also tell `run` to map SSH to it.
+
+```
+$ export COREOS_ASSEMBLER_CONTAINER_RUNTIME_ARGS='-p 2222:2222'
```

Comment:
This only applies if you're not already inside `cosa shell` keep in mind.

(Personally I still think we should push strongly towards documenting running everything inside `shell`)

Reply:
> This only applies if you're not already inside `cosa shell` keep in mind.

I think you still need this even if you are using `cosa shell` since we don't default to using `--net=host`. I guess it depends on if you are sshing from inside the container or not. 

> 
> (Personally I still think we should push strongly towards documenting running everything inside `shell`)

I think I disagree, but maybe we should open a separate issue to discuss.

Reply:
>  I guess it depends on if you are sshing from inside the container or not.

Exactly, I live inside my dev containers and often ssh from them.